terraform/environments/oasys/locals_test.tf

```diff
@@ -40,28 +40,26 @@ locals {
     }
 
     baseline_ec2_autoscaling_groups = {
-      # webserver = {
-      #   autoscaling_group = {
-      #     desired_capacity    = 1
-      #     max_size            = 2
-      #     vpc_zone_identifier = module.environment.subnets["private"].ids
-      #   }
-      #   autoscaling_schedules = module.baseline_presets.ec2_autoscaling_schedules.working_hours
-      #   config = merge(module.baseline_presets.ec2_instance.config.default, {
-      #     ami_name = "base_rhel_8_5_*"
-      #   })
-      #   ebs_volume_config = null
-      #   ebs_volumes       = null
-      #   instance = merge(module.baseline_presets.ec2_instance.instance.default, {
-      #     vpc_security_group_ids = ["private"]
-      #   })
-      #   lb_target_groups = null
-      #   ssm_parameters   = null
-      #   tags = {
-      #     os-type = "Linux"
-      #   }
-      #   user_data_cloud_init = module.baseline_presets.ec2_instance.user_data_cloud_init.ssm_agent_and_ansible
-      # }
+      webserver = {
+        autoscaling_group = {
```

Comment:
add these autoscaling group settings in baseline presets and then call that - it will be reused a lot